src/js/utils/themeDocUtils.js

```diff
@@ -74,6 +74,14 @@ export const themeDocUtils = {
       defaultValue: 'focus',
     },
   },
+  iconColor: {
```

Comment:
why should `iconColor` be part of `themeDocUtils.js`? is it being used for other components?

Reply:
Its being used in Carousel so I can go in and change that to use the themeDocUtils I believe those those are the only two using it Carousel and Calendar

Reply:
Sounds good, let's do it then.

Reply:
Ill do in another PR 👍

Reply:
I think you can add it on this PR, so the generic iconColor will be used for more than a single component.